bin/oref0-get-profile.js

```diff
@@ -18,7 +18,7 @@
 
 var generate = require('oref0/lib/profile/');
 function usage ( ) {
-        console.log('usage: ', process.argv.slice(0, 2), '<pump_settings.json> <bg_targets.json> <insulin_sensitivities.json> <basal_profile.json> [<preferences.json>] [<carb_ratios.json>]');
+        console.log('usage: ', process.argv.slice(0, 2), '<pump_settings.json> <bg_targets.json> <insulin_sensitivities.json> <basal_profile.json> [<preferences.json>] [model.json] [<carb_ratios.json>]');
```

Comment:
This will break everyone passing carb_ratios until they modify their loop configs. Need to instead add it as the last argument, or use yargs to make it a non-positional one.